Source/Particles/MultiParticleContainer.cpp

```diff
@@ -468,9 +468,21 @@ MultiParticleContainer::Evolve (int lev,
         if (rho) { rho->setVal(0.0); }
         if (crho) { crho->setVal(0.0); }
     }
+
+    // Get current iteration for use in supercycling
+    auto& warpx = WarpX::GetInstance();
+    int iter = warpx.getistep(0);
```

Comment:
Since this is called from WarpX, the `iter` could be passed in as an argument (the same way the `t` and `dt` are).